test/exectests.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import sys
+
+cmd = ["pytest"] + sys.argv[1:]
+print("EXECUTING:", cmd)
+
+process = subprocess.Popen(
+    cmd, bufsize=1, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+)
+for line in iter(process.stdout.readline, ""):
+    print(line, end="")
+    sys.stdout.flush()
+process.wait()
+sys.exit(process.returncode)
```

Comment:
what is the purpose of this file?

Reply:
There's no other way on my system to execute the tests: it always fail to import the QT .dlls, even after adding them to the PATH, and setting some QT envars. That's why I've mocked them on conftest.py.
Beside that, the only possibility of running pytest is from a Python application (like the regular WADAS), which is exactly why this extra script exists.
Anyway, I've removed both from the official code: I'll manually add them on my local configuration when I need to execute the test cases.
Only one request: I'd like to add exectests.py on .gitignore, so that I can keep at least this file on my local config, without pushing to the repo, if it's ok for you.